app/views/course/assessment/answer/_guided.html.slim

```diff
@@ -0,0 +1,2 @@
+- if answer.attempting?
+  = base_answer_form.button :button, t('common.submit'), value: answer.id, name: 'submit_answer'
```

Comment:
@lowjoel button helpers translation might not be good for this case, because answer is a nested attribute and the translation key is actually `helpers.submit.submission[answers_attributes][0].update`.  So I change to use `common.submit` instead.